### ISSUE TITLE
Admin-UI - Link Existing Relationship Dropdowns Don't Expand

### ISSUE THREAD

**Issue by artgibson** [keystonejs/keystone#5500]
## Describe the bug

When using the Web Admin UI to add an Existing Relationship to your model the drop down does not expand to the content and is unreadable. 

## To Reproduce

Steps to reproduce the behaviour. Please provide code snippets or a repository:

1. Setup a scheme with a 2 models , Model A has a relationship with Model B
2. Create data in the Web Admin UI for Model B
3. Create an entry for Model A , and click "Link existing Model B"
4. See issue with drop-downs not expanding to content

## Expected behaviour

Expected that after clicking "Link Existing Model B" the dropdown would expand to size of content.

## Screenshots

<img width="859" alt="select_notexpanded" src="https://user-images.githubusercontent.com/332115/115437765-9504e300-a1da-11eb-919e-5854f1496f15.png">

## System information

- OS: Mac OSX Catalina 10.15.4
- Browsers: Firefox 87.0,  Chrome 89.0, Safari 13.1.1

**Comment by artgibson** [comment]
this is for "@keystone-next/keystone": "15.0.0"

**Comment by bladey** [comment]
Hi @artgibson, thanks for raising this! We'll look into this shortly and get back to you.

**Comment by bladey** [comment]
Hi @artgibson,

Could you please replicate this in a [CodeSandbox fork](https://codesandbox.io/s/github/keystonejs/keystone/tree/master/examples-next/sandbox) and link it here or provide your schema if you're not able to provide one.

Thanks!

**Comment by artgibson** [comment]
@bladey - thanks, https://codesandbox.io/s/jnsms , on further review I believe the issue happens when the ui is set to "displayMode: "cards".

```
instruments: relationship({
  ref: "Instrument.musicians",
  ui: {
    displayMode: "cards",
    cardFields: ["name"],
    inlineEdit: { fields: ["name", "description"] },
    linkToItem: true,
    inlineConnect: true,
    inlineCreate: { fields: ["name", "description"] }
  },
  many: true
})
```

**Comment by bladey** [comment]
@artgibson thank you! We'll have a look.

**Comment by mitchellhamilton** [comment]
This was fixed by #5855
